### PR TITLE
Add first Navbar.

### PR DESCRIPTION
**Pull request by Tanabodee-Yambangyang**

### CODE REVIEW

**Review by jindasy — APPROVED** [review]
GOOD ka, OK to merge !👍🏼

File: frontend/fuiyoh/src/styles/NavBar.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  .topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+    width: 2000px;
```
Comment (GToidZ) [inline frontend/fuiyoh/src/styles/NavBar.css]: Do you need this?

**Review by GToidZ — CHANGES_REQUESTED** [review]
Need fixes.

File: frontend/fuiyoh/src/styles/NavBar.css (hunk 1 above)
Comment (GToidZ) [reply frontend/fuiyoh/src/styles/NavBar.css]: Line 13 has width of 2000px. A little bit overkill, isn't it?

**Review by GToidZ — APPROVED** [review]
Nice. I think it's good for merge now.

**Pull request merged**